src/openforms/variables/tests/test_validators.py

```diff
@@ -0,0 +1,206 @@
+from string import ascii_letters
+from typing import Any, Literal, TypeAlias
+
+from django.test import SimpleTestCase
+
+from hypothesis import given, strategies as st
+
+from ..validators import HeaderValidator, ValidationError
+
+FIELD_NAME_ALPHABET = "!#$%&'*+-.^_`|~0123456789" + ascii_letters
+VCHAR = "".join((chr(i) for i in range(0x21, 0x7F)))
+OBS_TEXT = "".join((chr(i) for i in range(0x80, 0x100)))
+SP = "\x20"
+HTAB = "\x09"
+CR = "\x0d"
+LF = "\x0a"
+NUL = "\x00"
+FIELD_VALUE_ALPHABET = "".join((VCHAR, OBS_TEXT, SP, HTAB))
+
+
+JSONValue: TypeAlias = (
```

Comment:
JSONValue is defined in `openforms.typing` already (though I forgot `bool`, it seems)